lib/rails_charts/base_chart.rb

```diff
@@ -71,6 +71,7 @@ def js_code
 
             window.addEventListener('load', init_#{chart_id});
             window.addEventListener('turbo:load', init_#{chart_id});
+            window.addEventListener('turbo:frame-load', init_#{chart_id});
```

Comment:
I don't think it will work like this, since we need to make sure that in case page has multiple turbo frames, we are not triggering this call for already rendered charts.